### DISCUSSION TITLE
SAE 2.0.6 - Node Red Dashboard erstellen

### DISCUSSION THREAD

**Discussion by alexb2506** [camueller/SmartApplianceEnabler#361] [Allgemein]
Hallo,
vielen Dank erst einmal für die Tolle Software. 

Ich bin recht neu auf diesem Gebiet. Ich bin nun von 1.6.2 auf 2.0.6 umgestiegen. Nun sehe ich hier die Option Dashboards mit Node Red zu erstellen. Kenne mich jedoch nicht wirklich "noch" damit aus. 

Mich würde interessieren wer schon welche erstellt hat und welche Daten Ihr im Dashboard abbildet.
Bekommt Ihr auch die Daten des Ertrages abgebildet?

Wäre es möglich Beispiele hier bereitzustellen?
VG Alex

**Comment by camueller** [comment]
Die Dokumentation hast Du gefunden? https://github.com/camueller/SmartApplianceEnabler/blob/2.0/doc/NodeRED_DE.md

Ab dem Kapitel [Export der Flows](https://github.com/camueller/SmartApplianceEnabler/blob/2.0/doc/NodeRED_DE.md#export-der-flows)
ist beschrieben, wie Du ein Standard-Dashboard aus dem SAE nach NodeRED übernehmen kannst. Vielleicht reicht Dir das ja schon. Und wenn nicht zeigt es Dir, wie Du für Anpassungen oder eigene Dashboards vorgehen musst. Vorher empfehle ich Dir aber ein paar Tutorials zu NodeRED :-)

**Reply by alexb2506** [reply]
Ja, das habe ich komplett durchgearbeitet und erstellt.
Mir ging es mehr drum, hier sind mit Sicherheit experten dabei die richtig tolle und interessante Boards erstellt haben, es zu sehen.

**Reply by camueller** [reply]
Da hatte ich mir auch mehr Feedback erhofft - bisher leider nichts. In dem "Standard-Dashboard" habe ich einfach zweckmäßige Informationen zusammengestellt. Manches kann man sicher besser machen ...

**Comment by alexb2506** [comment]
Eventuell ist jemand bereit sein Dashboard zu teilen. Würde mich sehr darüber freuen.
VG
